src/comm/LinkManager.cc

```diff
@@ -221,7 +220,7 @@ SharedLinkInterfacePtr LinkManager::sharedLinkInterfacePointerForLink(LinkInterf
         }
     }
 
-    qWarning() << "LinkManager::sharedLinkInterfaceForLink returning nullptr";
```

Comment:
This should really stay as a qWarning since it indicates an internal error. And qWarning will also fail units tests if it hits. Super useful to have it pop all the time not behind a category flag just like it popped for this crash.